{"name": "test_MessagesApi01_send_login_code", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <TestCase.Notification.MessagesApi.test_MessagesApi.TestMessagesApi object at 0x00000291D440A4F0>\n\n    @allure.story('发送登陆短信验证码')\n    def test_MessagesApi01_send_login_code(self):\n        response = zhuorui('消息通知', '发送登陆短信验证码')\n        assert response.status_code == 200\n        if response.json().get(\"code\") == \"000000\":\n            assert_data(response, '000000', 'ok')\n        elif response.json().get(\"code\") == \"030002\":\n>           assert_data(response, '030002', '当天短信验证码超过次数')\n\ntest_MessagesApi.py:23: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse = <Response [200]>, code = '030002', msg = '当天短信验证码超过次数'\n\n    def assert_data(response: object, code: object, msg: object) -> object:\n        assert code == response.json().get(\"code\")\n>       assert msg == response.json().get(\"msg\")\nE       AssertionError\n\n..\\..\\..\\Common\\assertapi.py:3: AssertionError"}, "start": 1606818619255, "stop": 1606818619318, "uuid": "452a5fdb-82c5-421b-9cf1-163a6824f981", "historyId": "c9efa658df8fa1a41e2bc31950af6493", "testCaseId": "fe762a337112db812c742b1ac8dc1f48", "fullName": "TestCase.Notification.MessagesApi.test_MessagesApi.TestMessagesApi#test_MessagesApi01_send_login_code", "labels": [{"name": "story", "value": "发送登陆短信验证码"}, {"name": "feature", "value": "消息通知"}, {"name": "parentSuite", "value": "TestCase.Notification.MessagesApi"}, {"name": "suite", "value": "test_MessagesApi"}, {"name": "subSuite", "value": "TestMessagesApi"}, {"name": "host", "value": "DESKTOP-KBU1G8M"}, {"name": "thread", "value": "113596-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.Notification.MessagesApi.test_MessagesApi"}]}